### PR TITLE
Supports inserting commas into quoted field

### PR DESCRIPTION
**Pull request by theescapers**
Supports inserting commas into quoted field, but unquoted behaviour remains the same

Also prevent crash on deleting last line

Fixes # .

### Checklist

- [ ] I have included examples or tests
- [ ] I have updated the change log
- [ ] I am listed in the CONTRIBUTORS file
- [ ] I have cleaned up the commit history (use rebase and squash)

### Changes proposed in this pull request:

-
-
-

### CODE REVIEW

File: src/Orc.CsvTextEditor/Services/CsvTextEditorInstance.cs [hunk 1]
```diff
@@ -444,7 +444,15 @@ private void ClearSelectedText()
 
             _textEditor.SelectionLength = 0;
             UpdateText(text);
-            _textEditor.CaretOffset = selectionStart;
+
+            if (text.Length < selectionStart)
+            {
+                _textEditor.CaretOffset = text.Length;
+            } else
```
Comment (GeertvanHorrik) [inline src/Orc.CsvTextEditor/Services/CsvTextEditorInstance.cs]: else on newline

**Comment by GeertvanHorrik** [comment]
Please run the unit tests as well, there are public API changes in this PR.

**Comment by GeertvanHorrik** [comment]
Did you run the unit tests to approve the API changes?

**Comment by theescapers** [comment]
Hey Geert, was just able to run the tests now, all ran without issue.

**Comment by GeertvanHorrik** [comment]
thanks!

**Pull request merged**